SwiftCompilerSources/Sources/SIL/BasicBlock.swift

```diff
@@ -77,6 +77,10 @@ final public class BasicBlock : CustomStringConvertible, HasShortDescription, Ha
     }
   }
 
+  public var hasKnownTerminator: Bool {
```

Comment:
What does `hasKnownTerminator` mean?
You should remove this function and use `block.termInst.isFunctionExiting || block.termInst is UnreachableInst` in the caller

Reply:
Removed